.github/workflows/lint-test.yaml

```diff
@@ -12,14 +12,14 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Helm
-        uses: azure/setup-helm@v3
+        uses: azure/setup-helm@v4
```

Comment:
Helm can be removed. Already installed on official GitHub runners

Reply:
https://github.com/actions/runner-images/blob/main/images%2Fubuntu%2FUbuntu2404-Readme.md